howso/visuals/visuals.py

```diff
@@ -581,3 +581,48 @@ def plot_interpretable_prediction(
 
     if len(figures) == 1:
         return figures[0]
+
+
+def plot_fairness_disparity(
+    fairness_results: dict,
+    ref: str,
+    threshold: float = 0.75,
+    x_rotate: bool = False
```

Comment:
Can this be `bool | float = False`? That way the user can also specify the specific degrees they'd like to rotate the label?

Reply:
The name may be better suited as `x_tickangle` as well, to match the name of the parameter in Plot.y